app/javascript/components/bootcamp/JikiscriptExercisePage/Header/Header.tsx

```diff
@@ -39,7 +41,7 @@ function _Header() {
         </div>
       </div>
       <div className="ml-auto flex items-center gap-8">
-        <CustomFunctionsButton />
+        {exercise.language === 'jikiscript' && <CustomFunctionsButton />}
```

Comment:
Maybe you could just add a `isJikiscript` property (or similar) so we're not having to do the same comparison in lots of places?